internal/query_builder.go

```diff
@@ -183,7 +181,7 @@ func ToWorkflowStatus(statusString string) (WorkflowStatus, error) {
 	status := WorkflowStatus(strings.ToUpper(statusString))
 	switch status {
 	case WorkflowStatusOpen, WorkflowStatusClosed, WorkflowStatusCompleted,
-		WorkflowStatusFailed, WorkflowStatusCanceled, WorkflowStatusTerminated,
+		WorkflowStatusFailed, WorkflowStatusCanceled,
```

Comment:
you can still leave it here in the mapping function.

Reply:
I will readd it later. removed it to be sure that the status can not be used in any situation at all.